spec/unit/migration_spec.rb

```diff
@@ -101,7 +101,7 @@ class MigrateFoo < Migration; end
       Migration.migrate_description(store, "v3_description")
       description = SystemDescription.load("v3_description", store)
       expect(description.format_version).to eq(SystemDescription::CURRENT_FORMAT_VERSION)
-      expect(captured_machinery_output).to include("is up to date, no upgrade necessary")
+      expect(captured_machinery_output).to include("no upgrade necessary")
```

Comment:
If you change the message to begin with an uppercase letter you need to adapt this line, too.